web3deployment/deployer.js

```diff
@@ -272,7 +273,22 @@ async function main() {
                                                             tokens[i],
                                                             ethAddress,
                                                             true));
+
+    var srcString1 = web3.utils.sha3("src token " + (2*i).toString());
+    var destString1 = web3.utils.sha3("dest token " + (2*i).toString());
+    var srcString2 = web3.utils.sha3("src token " + (2*i + 1).toString());
+    var destString2 = web3.utils.sha3("dest token " + (2*i + 1).toString());
+
+    await sendTx(networkContract.methods.setInfo(srcString1, ethAddress));
+    await sendTx(networkContract.methods.setInfo(destString1, tokens[i]));
+    await sendTx(networkContract.methods.setInfo(srcString2, tokens[i]));
+    await sendTx(networkContract.methods.setInfo(destString2, ethAddress));
   }
```

Comment:
I assume set info required also for network state - for the UI

Reply:
One we define it, yes.